src/audio-worklet/free-queue/example/js/freequeue.mjs

```diff
@@ -0,0 +1,170 @@
+/**
```

Comment:
Please follow JS coding style:
https://google.github.io/styleguide/jsguide.html

Or you can simply use Google ESLint in the local dev setting:
https://github.com/google/eslint-config-google